python/cudf/cudf/tests/test_series.py

```diff
@@ -772,6 +775,40 @@ def test_round_nan_as_null_false(series, decimal):
     assert_eq(result, expected, atol=1e-10)
 
 
+@pytest.mark.parametrize(
+    "data, dtype, decimals, expected_half_up, expected_half_even",
+    [
+        # Decimal32Dtype Test Cases
+        ([1.234, 2.345, 3.456], Decimal32Dtype(precision=5, scale=3), 2, [1.23, 2.35, 3.46], [1.23, 2.34, 3.46]),
+        ([1.234, 2.345, 3.456], Decimal32Dtype(precision=5, scale=3), 0, [1.0, 2.0, 3.0], [1.0, 2.0, 3.0]),
+        ([1.234, 2.345, 3.456], Decimal32Dtype(precision=5, scale=3), 3, [1.234, 2.345, 3.456], [1.234, 2.345, 3.456]),
+        # Decimal64Dtype Test Cases
+        ([1.234567, 2.345678, 3.456789], Decimal64Dtype(precision=10, scale=6), 4, [1.2346, 2.3457, 3.4568], [1.2346, 2.3457, 3.4568]),
+        ([1.234567, 2.345678, 3.456789], Decimal64Dtype(precision=10, scale=6), 2, [1.23, 2.35, 3.46], [1.23, 2.35, 3.46]),
+        ([1.234567, 2.345678, 3.456789], Decimal64Dtype(precision=10, scale=6), 6, [1.234567, 2.345678, 3.456789], [1.234567, 2.345678, 3.456789]),
+    ],
+)
+def test_series_round_decimal(data, dtype, decimals, expected_half_up, expected_half_even):
+    ser = cudf.Series(data).astype(dtype)
+
+    # Half-up rounding
+    result_half_up = ser.round(decimals=decimals, how="half_up").astype(dtype)
```

Comment:
Can you parametrize over `"half_up"` and `"half_even"` too? Also could you remove the comments and prints here?

Reply:
Thank you for the feedback.

Regarding the suggestion to parametrize over "half_up" and "half_even":

Currently, the round method directly accepts "half_up" and "half_even" as arguments for the how parameter. Adding parameterization might make the test more complex and less clear since the rounding method is already a direct input. Could you clarify the intended benefit of this change, or if there is a specific scenario you'd like to address by parameterizing "how"?
Regarding comments and print statements:

I have removed all comments and print statements from the test function as suggested.
Please let me know if there are further refinements you'd like me to make.